src/test/java/org/elasticsearch/transport/netty/NettyTransportTests.java

```diff
@@ -76,6 +79,7 @@ public void testThatConnectionFailsAsIntended() throws Exception {
             fail("Expected exception, but didnt happen");
         } catch (ElasticsearchException e) {
             assertThat(e.getMessage(), containsString("MY MESSAGE"));
+            assertThat(channelProfileName, is("default"));
```

Comment:
You should be able to use the "default" constant you defined here so the test will still pass if it's changed in the future.